mkt/search/tests/test_views.py

```diff
@@ -1414,7 +1414,8 @@ def make_homescreen(self):
             device_type=mkt.DEVICE_GAIA.id)
         self.homescreen.update(categories=['health-fitness', 'productivity'])
         self.homescreen.update_version()
-        self.refresh(('webapp', 'website', 'extension'))
+        HomescreenIndexer.index_ids([self.homescreen.pk], no_delay=True)
```

Comment:
Actually `refresh()` should call `_send_tasks()` so why do we need the explicit indexation ?